gdx/src/com/badlogic/gdx/maps/tiled/renderers/BatchTiledMapRenderer.java

```diff
@@ -186,7 +186,11 @@ public void renderObject (MapObject object) {
 	@Override
 	public void renderImageLayer (TiledMapImageLayer layer) {
 		final Color batchColor = batch.getColor();
-		final float color = Color.toFloatBits(batchColor.r, batchColor.g, batchColor.b, batchColor.a * layer.getOpacity());
+		final Color combinedTint = layer.getCombinedTintColor();
+		// For image layer rendering, multiply all by alpha except opacity
+		final float color = Color.toFloatBits(batchColor.r * (combinedTint.r * combinedTint.a),
```

Comment:
It looks like this is premultiplying alpha; if Tiled defaults to that or doesn't offer the option to change the option, this is probably good as-is. I don't think anything else in libGDX defaults to premultiplied alpha, but Tiled isn't in libGDX. And there's also the mostly-theoretical concern that none of the renderers are supposed to be Tiled-specific unless they mention they're for TMX maps... @raeleus ! You have been summoned! Since you have used other types of non-TMX maps with libGDX, would this change affect your usage in any undesirable way?

Reply:
The way I do level design is so far removed from TMX support that there is nothing you can do here that would affect my approach.
With that said, there is a basic assumption across the board that with any editor you are not going to have a 1:1 relationship between what you see in the editor and what the game is actually going to look like. I do appreciate the sentiment of this PR to bring rendering as close as possible. I don't think we should make technical choices like pre-multiplied the default.
Anyway, ignore me. I use busted old Ogmo and complain about it every time. I don't use Tiled and likely never will. Next thing for me is to learn Hyperlap as an alternative. Someday.

Reply:
Yeah, it sounds like you two are in agreement 6 minutes apart! Since PNGs act like libGDX normally does (without PMA), we probably shouldn't force it on, but having it as an option (like how particle effects have it) might make sense to make things act like how JPG acts in Tiled. I don't think Tiled will ever look exactly like a game will at runtime in all circumstances, but the tints can at least get closer.

Reply:
I made one final update to the get the functionality into parity with Tiled.
For what its worth, it now all works as it should. Image layers with jpgs, pngs, gifs. Since Tiled supports pngs and gifs for tint color transparency I simply check in the TiledMapImageLayer and then handle it in the renderer. Modified all the tests to have pngs now and all works and matches as it should. If it's too convoluted I could revert it all. But at least now it all matches.

Thanks for all the help and back and forth on these forsaken TiledMap Issues.